translate/convert/po2flatxml.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+
+"""Convert Gettext PO localization files to flat XML files.
+
+TODO: examples/usage
+"""
+
+from translate.convert import convert
+from translate.storage import flatxml, factory
+
+
+class po2flatxml(object):
+    """Convert to a single PO file to a single XML file, optionally
+    applying modifications to a template file instead of creating
+    one from scratch based on input parameters.
+    """
+
+    def __init__(self, inputfile, outputfile, templatefile=None,
+                 root="root", value="str", key="key", ns=None, indent=2):
+        """Initialize the converter."""
+        self.inputfile = inputfile
+        self.outputfile = outputfile
+        self.templatefile = templatefile
+
+        self.value_name = value
+        self.key_name = key
+        self.namespace = ns
+
+        indent_chars = None
+        if indent > 0:
+            indent_chars = (" " * indent)
+
+        self.source_store = factory.getobject(inputfile)
+        self.target_store = flatxml.FlatXMLFile(templatefile,
+                                                root_name=root,
+                                                value_name=value,
+                                                key_name=key,
+                                                namespace=ns,
+                                                indent_chars=indent_chars)
+
+    def convert_unit(self, unit):
+        """Convert a source format unit to a target format unit."""
+        target_unit = flatxml.FlatXMLUnit(source=None,
```

Comment:
Nit: Don't put the target unit class here, instead specify it as a `TargetUnitClass` class attribute above.